bullet_train-internationalization/bullet_train-internationalization.gemspec

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_relative "lib/bullet_train/internationalization/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "bullet_train-internationalization"
+  spec.version = BulletTrain::Internationalization::VERSION
+  spec.authors = ["Andrew Culver", "Gabriel Zayas"]
+  spec.email = ["andrew.culver@gmail.com", "g-zayas@hotmail.com"]
+  spec.homepage = "https://github.com/bullet-train-co/bullet_train"
+  spec.summary = "A gem for housing Bullet Train's standard localizations."
+  spec.description = spec.summary
+  spec.license = "MIT"
+
+  # spec.metadata["allowed_push_host"] = "TODO: Set to your gem server 'https://example.com'"
+
+  spec.metadata["homepage_uri"] = spec.homepage
+  spec.metadata["source_code_uri"] = spec.homepage
+  spec.metadata["changelog_uri"] = spec.homepage
+
+  # Specify which files should be added to the gem when it is released.
+  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
+  spec.files = Dir.chdir(__dir__) do
+    `git ls-files -z`.split("\x0").reject do |f|
+      (File.expand_path(f) == __FILE__) || f.start_with?(*%w[bin/ test/ spec/ features/ .git .circleci appveyor])
+    end
+  end
+  spec.bindir = "exe"
+  spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
+
+  spec.add_development_dependency "rails", "~> 7.0.0"
```

Comment:
@jagthedrummer I've cleaned up the PR a bit, but the only thing I'm not sure about right now is the gem versions that are in here and the Gemfile which are set explicitly. I don't <i>think</i> they should be, but just wanted to double check what you think about it.

Reply:
@gazayas I think we probably want to match whatever version constraints the other `core` gems are using.

Reply:
@jagthedrummer Alright, what I've got here matches what we have in `bullet_train.gemspec` and `bullet_train-roles.gemspec`.